views/forms/form_match.php

```diff
@@ -22,39 +21,134 @@
 // Grab info passed from hidden for mfields
 $firstName = ucwords(trim($_POST['firstname']));
 $lastName = ucwords(trim($_POST['lastname']));
-
 // Query the db to find any duplicate entries
 $db->prepare("get-duplicate-data", "SELECT DISTINCT ON (peopleid) *
                                     FROM ParticipantModal
                                     WHERE firstname = LOWER($1) AND lastname = LOWER($2) LIMIT 10");
```

Comment:
I think instead of saving everything as lower case in the database, just make this query:
```sql
SELECT DISTINCT ON (peopleid) *
FROM ParticipantModal
WHERE LOWER(firstname) = LOWER($1) AND LOWER(lastname) = LOWER($2) LIMIT 10;